web-server/v0.4/src/components/PageHeader/index.test.js

```diff
@@ -10,6 +13,21 @@ const routerData = {
   },
 };
 
+const { TabPane } = Tabs;
+const mockProps = {
+  selectedControllers: undefined,
```

Comment:
Can we consider initializing `selectedControllers` as the same default data type/structure we specify in the Redux store?